File: TaggyAppBackend/TaggyAppBackend.Api/Models/Dtos/GroupUser/GetGroupUserDto.cs
using TaggyAppBackend.Api.Models.Enums;

namespace TaggyAppBackend.Api.Models.Dtos.GroupUser;

public class GetGroupUserDto : GetDto
{
    public string UserId { get; set; } = null!;
    public string Name { get; set; } = null!;
    public string Email { get; set; } = null!;
    public GroupRole Role { get; set; }
}

File: TaggyAppFrontend/src/app/models/dtos/groupUser/getGroupUserDto.ts
import { GroupRole } from '../../enums/groupRole';
import { GetDto } from '../getDto';
import { GetDtoInterface } from '../getDtoInterface';

export class GetGroupUserDto implements GetDtoInterface {
  get id(): string {
    return this.userId;
  }
  userId!: string;
  name!: string;
  email!: string;
  role!: GroupRole;
  createdAt!: Date;
}
